libsolidity/analysis/ReferencesResolver.cpp

```diff
@@ -283,4 +299,58 @@ void ReferencesResolver::operator()(yul::VariableDeclaration const& _varDecl)
 		visit(*_varDecl.value);
 }
 
+void ReferencesResolver::resolveInheritDoc(StructuredDocumentation const& _documentation, StructurallyDocumentedAnnotation& _annotation)
+{
+	switch (_annotation.docTags.count("inheritdoc"))
```

Comment:
Do you think using a switch makes this easy to understand? :)

Reply:
At first I wasn't but @christianparpart  convinced me and now I think he's right.

Reply:
So you both think that upon reading the world `default` it is fine to be forced to first check what the other cases are to arrive at the conclusion that the default case is "more than one"?

Reply:
To be fair, the position of the `default:` I'd rather have had at the end of the cases, but the compiler didn't like that

Reply:
Ah, now it does. Dunno what the problem was